### PR TITLE
Fix model create

### PR DESCRIPTION
**Pull request by dougmrqs**
Many tests were skipped and commented why. Most of them are CockroachDB limitations.
Added some limitations noted on tests to README.

### CODE REVIEW

File: .github/workflows/test_names.txt [hunk 1]
```diff
@@ -22,4 +22,17 @@ should change columns
...
+doesn't allow case-insensitive duplicated records using CITEXT
+allows the creation of a TSVECTOR field
+TSVECTOR only allow string
+doesn't allow duplicated records with unique function based indexes
+sets auto increment fields
+should return autoIncrement primary key \(create\)
+several concurrent calls
+should error correctly when defaults contain a unique key and the where clause is complex
+should work with multiple concurrent calls
+should not deadlock with concurrency duplicate entries and no outer transaction
```
Comment (RichardJCai) [inline .github/workflows/test_names.txt:38]: nit: small request, add EOL so we don't see a diff for the last line when we edit it.

File: tests/model_create_test.js [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+        return User.findOrCreate({
+          where: {
+            email: 'unique.email.1@sequelizejs.com',
+            companyId: 2
+          }
+        });
+      }));
+    });
+
+    // Reason: Transaction stuff. Usually crashes Sequelize and times out tests by 60 seconds.
```
Comment (RichardJCai) [inline tests/model_create_test.js:119]: I think these limitations are worth documenting in the read me.

**Review by RichardJCai — APPROVED** [review]
Looks good, thanks.

File: tests/model_create_test.js (hunk 2 above)
Comment (dougmrqs) [reply tests/model_create_test.js:119]: I think they already are. I'll try to just explain it more clearly there.

File: .github/workflows/test_names.txt (hunk 1 above)
Comment (dougmrqs) [reply .github/workflows/test_names.txt]: I think the new regex will fix it. Good suggestion.

Comment (dougmrqs) [reply .github/workflows/test_names.txt]: UPDATE: The regex did not work. The best way is to refactor the ignored tests to a programmatic usage.

**Review by RichardJCai — APPROVED** [review]

**Pull request merged**